docs/resource/k8s.md

```diff
@@ -354,3 +354,69 @@ A CronJob creates Jobs on a repeating schedule.
 <!-- endsemconv -->
 
 [DocumentStatus]: https://opentelemetry.io/docs/specs/otel/document-status
+
+## Specify resource attributes using Kubernetes annotations
+
+All annotations with the `resource.opentelemetry.io/` prefix should be translated into the corresponding resource
+attributes.
+
+For example, the annotation `resource.opentelemetry.io/service.name` should be translated into the `service.name`
+attribute.
+
+## Service
+
+The following [service resource attributes](../attributes-registry/service.md) are recommended for Kubernetes services.
+
+There are different ways to calculate the service attributes.
+
+1. [Well-Known Labels](https://kubernetes.io/docs/reference/labels-annotations-taints/)
+2. Annotations on the pod template that have the `resource.opentelemetry.io/` prefix as described above
+3. A function of the Kubernetes resource attributes defined above
+
+This translation can typically be done by an OpenTelemetry Collector component like the
+[Kubernetes Attribute Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor).
+
+Tools offering this functionality should provide an opt-in flag for the use of well-known labels,
+since users may not be aware that their labels are being used for this purpose.
+
+Each attribute has a priority order for how it should be calculated as described below.
+
+### How `service.namespace` is calculated
+
+Choose the first value found:
+
+1. `pod.annotation[resource.opentelemetry.io/service.namespace]`
+2. `pod.label[app.kubernetes.io/part-of]` (well-known label
```

Comment:
Should we add k8s.namespace.name as a fallback here?

Reply:
good idea

@cyrille-leclerc wdyt?

Reply:
+1 to fallback to `k8s.namespace.name`.

Reply:
added